text/0026-triage-party.md

```diff
@@ -0,0 +1,61 @@
+
+## Motivation
+
+Keeping on top of all open issues, PRs and comments from the community and deciding which potential work to priortize is challenging. There is not an easy way to get visibility into the health of the community in terms of the age of open issues, issues requiring a response, and PRs awaiting review across all repositories.
+
+## Detailed Explanation
+
+The [Paketo project](https://paketo.io) is made up of many git repositories across the [paketo-buildpacks](https://github.com/paketo-buildpacks) and [paketo-community](https://github.com/paketo-community) Github orgs. These repositories contain implementation buildpacks, language family buildpacks, and auxiliary repositories related to managing the Paketo project and its community. **As of late April, 2021, there are 113 repositories across Paketo's two Github orgs.** At any given time each repository can have open issues in various stages of triage, and open pull requests requing the attention of language family maintainers and others in the community.
+
+## Rationale and Alternatives
+
+Triage party provides unified views, filters, and metrics across repositories that can be customized to the needs of the Paketo project and community. Triage party is open source and already used by several large projects including [Minicube](http://tinyurl.com/mk-tparty) and [skaffold](http://tinyurl.com/skaffold-tparty)
+
+While there are a number of comercial services that could allow us to more easily triage issue across repositories their primary purpose is more around providing unified Kanban boards (Something Triage Party could also do) and licening costs may not allow these comercial services to be shared widly with the community. Because Triage Party provides read only views across n repositories there are also no user accounts or permissions to administer.
+
+Paketo has an an existing custom built dashboard at https://dashboard.paketo.io which gives you an overview of issue counts and a unified list of open issues. While this dashboard is great as a quick way to glance at the state of repos it suffers from a few shrotcommings:
+
+ 1. It requires each user to provide their personal github token which is a barier to use.
+
+ 1. There is no included rules engine that can be easily customized via a configuration file. Changes to filtering and and other enhancments require additional engineering time.
+
+ 1. Given the breadth of the languages and buildpacks the Paketo project supports and the number of full time engineers known to be working on the project and current level of overal contributors, it's unlikley that the dashboard software will get much sustained development going forward.
+
+Finaly, Github search is somewhat useful and can be used to search for labels across repositories with some accuracy in a given Org. This approach, however, does not provide the at a gance overview of all repositories and metrics available by Triage Party.
```

Comment:
I would say one drawback is that for me Triage Party is a bit confusing to look at. Maybe I need to get used to it/use it more to get acquainted with it. If we're directly comparing it to the Paketo Dashboard, it's less user friendly than the dashboard IMO.